praw/reddit.py

```diff
@@ -481,8 +488,11 @@ def get(
             None).
 
         """
-        data = self.request("GET", path, params=params)
-        return self._objector.objectify(data)
+        try:
+            data = self.request("GET", path, params=params)
+            return self._objector.objectify(data)
+        except _RetryAgain:
+            return self.get(path, params)
```

Comment:
I'd rather avoid the recursion here. Could this instead be a loop? Something like

```
while True:
    try:
        data = ...
        return ...
    except _RetryAgain:
        pass
```

In addition, do we want to add some "sanity check" here to make sure we don't get stuck retrying over and over again?